sprint/prep-work.md

```diff
@@ -0,0 +1,35 @@
+# Sprint preparation
+
+## Checklist
+
+:::{list-table}
+:header-rows: 1
+:stub-columns: 1
+:widths: 1 4 1 2
+
+* - #
+  - Task
+  - Status
+  - Notes
+* - 1
+  - Have a GitHub account?
+  - {fas}`check;sd-text-success`
+  - 
+
+:::
+
+
+## Sprint day
+
+%TODO: Improve text%
+
+:::{tip}
+To communicate during the sprint, we will be using Discord. The [Using the Discord Server](https://youtu.be/w2A8SknM-68) introduction video contains a brief introduction to the Discord Data Umbrella server with some tips on how to navigate the application.
+:::
+
+### Pair programming
+
+Live Share is an extension from Visual Studio Code that allows you to collaboratively edit and debug with others in real-time. It allows you to instantly share your current project, edit snippets of code at the same time or follow someone’s cursor while they program. To learn more about Live Share, visit [Live Share with Visual Studio Code](https://youtu.be/WHOiljOYGVw).
+
+
+### Environment setup
```

Comment:
env setup is covered in https://pymc-data-umbrella--47.org.readthedocs.build/en/47/sprint/docstring_tutorial.html#pre-requisites (starts there and is a bit conflated with the docstring updates). We could try to split it into a different doc, but I am not sure it would be a good idea, I like that is is end-to-end tutorial with clear instructions and even shows the expected output.

Reply:
I think we can split the pre-requisites as that apply for any contribution, maybe including the clone the repo section, and refer to them in this tutorial.

Reply:
Do you think we could have 3 pages maybe?

1. git+conda+text editor
2. pr tutorial, we can update https://www.pymc.io/projects/docs/en/stable/contributing/pr_tutorial.html with the content from the current docstring tutorial
   - we can also have it here on this website for now and port it later to the pymc repo 
4. docstring tutorial: focused on editing docstrings only without going into the git/github workflow in its text but linking to the other two

Reply:
Done!